servatrice/src/servatrice_database_interface.cpp

```diff
@@ -291,11 +291,48 @@ bool Servatrice_DatabaseInterface::checkUserIsBanned(const QString &ipAddress, c
     }
 
     return
-        checkUserIsIpBanned(ipAddress, banReason, banSecondsRemaining)
-        || checkUserIsNameBanned(userName, banReason, banSecondsRemaining);
+        checkUserIsIpBanned(ipAddress, banReason, banSecondsRemaining) || checkUserIsNameBanned(userName, banReason, banSecondsRemaining) || checkUserIsIdBanned(clientId, banReason, banSecondsRemaining);
```

Comment:
We may actually want to not short circuit here - consider:
If a by-ip ban exists for 10 seconds, and a by-name ban exists for 20 seconds, you will tell the user they are banned for 20 seconds.

You don't need to address that in this PR I think, it's not a new behavior, but we should ticket it for future improvement